common/decklist.cpp

```diff
@@ -564,7 +564,7 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         // Filter out MWS edition symbols and basic land extras
         QRegExp rx("\\[.*\\]\\s?");
         line.remove(rx);
-        rx.setPattern("\\(.*\\)");
+        rx.setPattern("\\\\s?(.*\\)");
```

Comment:
This looks not correct.

>I am adding `\\s?` at the start of the regex.

You had the right thought, but it should look a bit different.
Did you try `rx.setPattern("\\s?\\(.*\\)");`?